src/App.js

```diff
@@ -1,66 +1,106 @@
-import React from 'react';
-import TodoList from "./components/TodoComponents/TodoList"
-import TodoForm from "./components/TodoComponents/TodoForm"
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
-  
+
   state = {
     tasks: [
       {
-        task: 'Organize Garage',
+        phrase: "Organize Garage",
         id: 1528817077286,
-        completed: true
+        completed: false
       },
       {
-        task: 'Bake Cookies',
+        phrase: "Bake Cookies",
         id: 1528817084358,
         completed: false
       }
     ],
-    taskInput:""
-  }
-  
-  changeHandler = e =>{
-    e.preventDefault()
+    taskInput: ""
+  };
+
+  // Takes user input as a value, assigns it as the name 
+  changeHandler = e => {
+    e.preventDefault();
     this.setState({
       [e.target.name]: e.target.value
-    })
+    });
+  };
```

Comment:
Great job defining this function as a universal input handler! This will make it so that you can use the same function to handle the changes of multiple inputs.